android/shared/src/commonMain/kotlin/io/rebble/cobble/shared/ui/view/home/watches/WatchesListItem.kt

```diff
@@ -0,0 +1,76 @@
+package io.rebble.cobble.shared.ui.view.home.watches
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import io.rebble.cobble.shared.data.WatchItem
+import io.rebble.cobble.shared.ui.common.RebbleIcons
+import io.rebble.cobble.shared.ui.viewmodel.WatchesListViewModel
+
+@Composable
+fun WatchesListItem(watch: WatchItem, viewModel: WatchesListViewModel) {
+    Row(
+            modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 16.dp, vertical = 24.dp)
+                    .clickable { viewModel.selectWatch(watch) },
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            if (watch.isConnected) {
+                Box(
```

Comment:
These `Box`es are identical except background color, I think its better to determine the color by checking `if (watch.isConnected)` and use a single box